app/src/main/java/com/matmoongi/screens/SearchScreen.kt

```diff
@@ -1,7 +1,95 @@
 package com.matmoongi.screens
 
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material3.CenterAlignedTopAppBar
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.material3.TopAppBarDefaults.centerAlignedTopAppBarColors
 import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import com.matmoongi.R
+import com.matmoongi.data.SearchRestaurant
+import com.matmoongi.restaurantCards.RestaurantCard
+import kotlinx.coroutines.flow.StateFlow
 
+@ExperimentalMaterial3Api
 @Composable
-fun SearchScreen() {
+fun SearchScreen(searchRestaurantsState: StateFlow<List<SearchRestaurant>>) {
```

Comment:
```suggestion
fun SearchScreen(searchRestaurants: List<SearchRestaurant>) {
```
state flow를 전달 하는 대신에 `List<SearchRestaurant>` 전달하는 것이 

1. preview를 작성하기에 유리하고
2. 의존점이 하나라도 줄어드는

장점이 있을 것 같습니다.

Reply:
네 저번에 상위 컴포넌트에 상태로 전달하지 않으면 recompose가 안되길래 미리 보내놔 봤는데 아직 상태 변화하는 게 없어서 UI 만들때는 말씀하신 대로 리스트로 전달하겠습니다.